my-project/src/LanguageBox.jsx

```diff
@@ -0,0 +1,108 @@
+import { useState, useEffect, useContext } from 'react';
+import { Octokit } from '@octokit/rest';
+import './App.css'; // Optional: For styling
+import { MyContext } from './MyContext';
+
+const LanguageBox = () => {
+  // const { myState, setMyState } = useContext(MyContext); // This state is created for setting the access token
+  const [languages, setLanguages] = useState([]);
+  const [selectedLanguage, setSelectedLanguage] = useState('');
+  const [repos, setRepos] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [message, setMessage] = useState('');
+  //setting the accesstoken in environment variable
+  const apiToken = import.meta.env.VITE_API_KEY;
+  console.log(apiToken);
+  const octokit = new Octokit({ auth: apiToken });
+
+  useEffect(() => {
+    const fetchLanguages = async () => {
```

Comment:
You can use it here for now, But keep remember that we need to make seprate folders and file structure to maintain the api calls and other stuff.

Reply:
ok